Source/RxSwift/fromAsync.swift

```diff
@@ -75,7 +75,7 @@ public enum FromAsyncError: Error {
 public extension PrimitiveSequenceType where Trait == SingleTrait {
     /**
      Transforms an async function that returns data or error through a completionHandler in a function that returns data through a Single
-     - The returned function will thake the same arguments than asyncRequest, minus the last one
+     - The returned function will take the same arguments as asyncRequest, minus the last one
```

Comment:
nice... Thanks
However, it looks like the code was changed without adding anything to the Changelog. you can add #trivial to the PR title or body.